config/policy.go

```diff
@@ -80,6 +80,11 @@ type Policy struct {
 	// value of any existing value of a given header key.
 	SetRequestHeaders map[string]string `mapstructure:"set_request_headers" yaml:"set_request_headers,omitempty"`
 
+	// RemoveRequestHeaders removes a collection of headers from downstream request.
```

Comment:
nit: RemoveRequestHeaders removes a collection of headers from `a` downstream request

Reply:
Fixed 👍